projects/burnr/src/index.tsx

```diff
@@ -1,8 +1,5 @@
-// workaround for parcel in order to work with async/await
-import "regenerator-runtime/runtime"
-
-import React from "react"
-import ReactDOM from "react-dom"
+import * as React from "react"
```

Comment:
Since React 17 there is [no need to import React for using JSX](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html), so con we please get rid of this import?

Reply:
This gives an error:
```
'React' refers to a UMD global, but the current file is a module. Consider adding an import instead
```